scripts/elasticsearch-old.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Install the old version of Elasticsearch in /tmp and run it.
+set -ev
+mkdir /tmp/elasticsearch-old
+wget -O - https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.6.2.tar.gz | tar xz --directory=/tmp/elasticsearch-old --strip-components=1
```

Comment:
The `wget` commands here produce a ton of log spam on Travis because they log progress for every 200KB of the file downloaded. That isn't a huge problem because Travis collapses the output by default, but I wonder if it might even be enough to slow down the script slightly.